.github/workflows/validate-pr.yml

```diff
@@ -0,0 +1,57 @@
+# Validates the format of a PR title and body. Because the title will become the default commit message on squash-merge,
+# it should follow the conventional commit format and be 70 characters long or fewer. Because the body will later be
+# used in creating docs-site release notes, it must have a summary paragraph (70+ characters) followed by `\n---`.
+
+name: "Validate PR"
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
```

Comment:
Does this flow really need to run on commits?

Reply:
That's there because we don't manually edit the release PR itself, so the validation check hangs up the release PR, requiring override of status checks when cutting a release. An alternative would be to ignore validation for the release branch pattern. I'll try that approach.

Reply:
Can this go into the release queue in github and get spat back out if it doesnt meet the criteria, instead of CI actions in the PR?

Reply:
Turns out if the workflow doesn't run on synchronize it leaves the check in an "expected" yellow state, preventing merge. Leaving it in so it stays green.

Reply:
@metal-messiah are you suggesting we switch to using a merge queue and run our checks there?

Reply:
I would like to see us use merge queues and run the checks there.

Reply:
After exploring merge queues, I don't think they will work in the way we would like, though they could warrant exploration outside this PR.